scriptExamples/python/extended_payload.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+"""
+Script to debug the "Scriptable Notifications" add-on for Thunderbird.
+
+This script receives the extended message from the add-on and writes it to a
+log file in the user's home directory. The file has the same basename as this
+script, but the suffix ".log".
+
+Copyright (C) 2022  Stephan Helma
```

Comment:
I don't think I can merge a file with a copyright in the MIT licensed project.

Reply:
Why is that so? The file is still under the MIT license. I always tend to include a copyright notice of all contributors to a file into that file itself – beside a short license line. The contributors still have the copyright of the code they provided but are happy to publish it under the project's license.

This approach also helps in keeping the copyright (and the license, if added) if it is distributed outside of the package. So I would suggest to include a license one-liner and your copyright line to each file …

Reply:
All right, I didn't know that.